logger/export_test.go

```diff
@@ -25,3 +25,12 @@ func GetLogger() Logger {
 
 	return logger
 }
+
```

Comment:
Do you think we want a dedicated type for the flags?

Reply:
nah. In practice they're not used, and should probably be unexported.

Reply:
in fact, I'll do that once this lands

Reply:
+1